common/src/main/java/com/wynntils/features/user/overlays/MiniMapOverlayFeature.java

```diff
@@ -136,6 +142,80 @@ private void rotateMapToPlayer(PoseStack poseStack, float centerX, float centerZ
             poseStack.translate(-centerX, -centerZ, 0);
         }
 
+        // TODO move most of the buffer builder code into RenderUtils, maybe separate each map border into different
+        // files to avoid insanity involving tx1/ty1/tx2/ty2 + remove magic constant
+        private void renderRectangularMapBorder(
+                PoseStack poseStack, float renderX, float renderY, float width, float height) {
+            Texture texture = null;
+            int grooves = 0;
+            int tx1 = 0;
+            int ty1 = 0;
+            int tx2 = 0;
+            int ty2 = 0;
```

Comment:
MapBorderType could have these?

Reply:
At least that is how we do this with health bar overlay

Reply:
Mhm that'd be better